lib/device_map.py

```diff
@@ -994,7 +995,8 @@
 x86_kvm = {'device_type': 'kvm',
            'templates': ['generic-x86-kernel-ci-boot-template.json',
                          'generic-x86-kernel-ci-kselftest-template.json',
-                         'generic-x86-kernel-ci-hackbench-template.json'],
+                         'generic-x86-kernel-ci-hackbench-template.json',
+                         'qemu-boot-template.jinja2'],
```

Comment:
Good question, but that was already covered in previous commit:
https://github.com/kernelci/lava-ci/pull/61